src/cfn-custom-resources/non-spa-app/index.ts

```diff
@@ -0,0 +1,73 @@
+// Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT-0
+
+import {
+    CloudFormationCustomResourceHandler,
+    CloudFormationCustomResourceResponse,
+    CloudFormationCustomResourceDeleteEvent,
+    CloudFormationCustomResourceUpdateEvent
+} from 'aws-lambda';
+import axios from 'axios';
+import s3SpaUpload from 's3-spa-upload';
+import { mkdirSync } from 'fs';
+
+
+interface Configuration {
+    BucketName: string;
+}
+
+
+async function uploadApp(action: 'Create' | 'Update' | 'Delete', config: Configuration, physicalResourceId?: string) {
+    if (action === 'Create' || action === 'Update') {
+        await s3SpaUpload(`${__dirname}/app`, config.BucketName);
```

Comment:
Since it is just a single file we're uploading (not a dir with nested dirs and such) we could also just use aws-sdk client S3. Don't need to mkDirSync then either. Feels a bit weird to use a lib called "spa upload" for something that is not a SPA :)

Sorry I think I recommended this to you in the first place

Reply:
It's a single file in its current form, but what if the user just wants to add files/folders to the app? Is there a separate way they would accomplish this? Don't disagree with the `s3SpaUpload` being a bit weird, but it seems to do the job.